qiskit/transpiler/__init__.py

```diff
@@ -40,629 +41,787 @@
 Qiskit has four pre-built transpilation pipelines available here:
 :mod:`qiskit.transpiler.preset_passmanagers`.  Unless the reader is familiar with
 quantum circuit optimization methods and their usage, it is best to use one of
-these ready-made routines.
-
+these ready-made routines. By default the preset pass managers into 6 stages:
```

Comment:
maybe "are split into 6 stages"?

Reply:
I updated this in: https://github.com/Qiskit/qiskit-terra/pull/9087/commits/a46e97de3261a7a394c0f63ff23f818f5936f49f